completions/bash/skopeo

```diff
@@ -155,13 +159,7 @@ _skopeo_skopeo() {
        --version -v
        --help -h
      "
-     commands=$( ${COMP_WORDS[@]:0:$COMP_CWORD} --generate-bash-completion )
-
-     case "$prev" in
-         $main_options_with_args_glob )
```

Comment:
Sure, this is not making things _worse_, but it also doesn’t seem quite the right thing.

Consider `skopeo --policy `<completion>: After this change, the code will fall through the `*)` case below and try to complete a command name; it should either use the `-*)` case or exit here, which, in both cases, results in no completions.

I think this should somehow use the `$options_with_args` instead. (But then again, the use of `$options_in_args` in  `_complete_` does not work either, AFAICS, because it is not the right syntax for alternatives in a `case`. Randomly checking some examples in `/usr/share/bash-completion`, `_init_completion -s` looks like a possible option, but also quite a bit more work than locally fixing warnings.)

Reply:
I updated it now to check for the options. Just using the variable doesn't work without some glob magic, so I opted for maintaining the list manually. I also had to adapt the logic for selecting the function to avoid the script believing that there's a function for the value of `--policy $value` for instance.